core-blocks/paragraph/edit.native.js

```diff
@@ -0,0 +1,65 @@
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Component,
+} from '@wordpress/element';
+import { parse } from '@wordpress/blocks';
+
+import { RichText } from '@wordpress/editor';
+
+const _minHeight = 50;
+
+class ParagraphBlock extends Component {
+	render() {
+		const {
+			attributes,
+			setAttributes,
+			//className,
+			style,
+		} = this.props;
+
+		const {
+			placeholder,
+		} = attributes;
+
+		return (
+			<View>
+				<RichText
+					content={ { contentTree: attributes.content, eventCount: attributes.eventCount } }
+					style={ style, [
+						{ minHeight: Math.max( _minHeight, attributes.aztecHeight !== null ? attributes.aztecHeight : 0 ) },
+					] }
+					onChange={ ( event ) => {
+						// Create a React Tree from the new HTML
+						const newParaBlock = parse( '<!-- wp:paragraph --><p>' + event.content + '</p><!-- /wp:paragraph -->' )[ 0 ];
+						setAttributes( {
+							...this.props.attributes,
+							content: newParaBlock.attributes.content,
+							eventCount: event.eventCount,
+						} );
+					}
+					}
+					onContentSizeChange={ ( event ) => {
+						setAttributes( {
+							...this.props.attributes,
+							aztecHeight: event.aztecHeight,
+						} );
+					}
+					}
+					placeholder={ placeholder || __( 'Add text or type / to add content' ) }
+					aria-label={ __( 'test' ) }
+				/>
+			</View>
+		);
+	}
+}
+
+const ParagraphEdit = function ParagraphEdit( { attributes, setAttributes } ) {
```

Comment:
We don't need this one, you can export `ParagraphBlock` directly. 

Aside: `const ParagraphEdit =` is obsolete in such code, it assigns the name to the function, but the function has already name specified.